src/kOS/Suffixed/BodyTarget.cs

```diff
@@ -122,6 +122,10 @@ private void BodyInitializeSuffixes()
                       new OneArgsSuffix<ScalarValue, Vector>(
                               AltitudeFromPosition,
                               "Interpret the vector given as a 3D position, and return its altitude above 'sea level' of this body."));
+            AddSuffix("GEOPOSITION",
```

Comment:
This suffix is hiding the suffix inherited from `orbitable`, with a very different behavior.  I think we should rename this new suffix to better represent that it is a different behavior from other suffix by the same name.  Maybe `childgeoposiiton` or `newgeoposition` or something like that.